### ISSUE TITLE
Throw an error on files that can not be provided from "content" node

### ISSUE THREAD

**Issue by l-johanson** [genodelabs/goa#33]
Analog to the extended validation of elements in the `<parent-provides>`-node in `runtime`-files worked on in #26 , I would like to extend the validation to throw an error on files from the `<content>`-node, which can not be provided.

For example the following `inexistent` file reference would not trigger a validation error in the `runtime`-file:
```
<content>
    <rom label="existent">
    <rom label="inexistent">
</content>
```

This would speed up debugging for users a lot and especially help eliminating first frustrations with the tooling.

Would you like to implement this or if not, are there any valuable hints you might give me concerning the implementation?

Kind regards,
Lars

**Comment by jschlatow** [comment]
Fixed by 86e0853

**Comment by jschlatow** [comment]
The commits entered the master branch as 82db891
